universal-login-contracts/contracts/ERC1077.sol

```diff
@@ -31,18 +31,19 @@ contract ERC1077 is KeyHolder, IERC1077 {
         OperationType operationType,
         bytes memory signatures) public view returns (bool)
     {
-        address signer = getSigner(
-            address(this),
-            to,
-            value,
-            data,
-            nonce,
-            gasPrice,
-            gasToken,
-            gasLimit,
-            operationType,
-            signatures);
-        return keyExist(bytes32(uint256(signer)));
+        address[] memory signers = new address[](signatures.length.div(65));
```

Comment:
We should add check for number of signatures to match required signatures, so:
- add new contract variable `uint requiredSignatures = 1` 
- add a private setter: `function setRequiredSignatures(value) onlyManagementKeyOrThisContract {...}`
- add a check at the beginning of the function: `require(requiredSignatures == signatures.length.div(65))`

Reply:
Perhaps in next PR - that might be a lot of additional testing.

Reply:
so for the moment it's better to not create the requiredSignature in this PR. right?